docs/apache-airflow-providers-ftp/operators/index.rst

```diff
@@ -0,0 +1,64 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+.. _howto/operator:FTPOperator:
+
+FTPOperator
+===============
+
+
+Use the FTPOperator to get or
+pull files to/from an FTP server.
+
+Using the Operator
+^^^^^^^^^^^^^^^^^^
+
+| **ftp_conn_id**: id referencing an FTP Connection **str**
+| **local_filepath**: local file path to get or put. **str**
+| **remote_filepath**: remote file path to get or put. **str**
+| **operation**: specify operation 'get' or 'put' **str** - Default: **PUT**
+| **create_intermediate_dirs**: create missing intermediate directories when copying from remote to local and vice-versa. **bool** - Default: **False**
```

Comment:
Just link to the class using `:class:` (search around for examples in `airflow/docs` I can't link one easily since Github will just render the rst and not deep link to the code line)  instead of hardcoding the current set of arguments here, this will quickly become out of date/sync.